### PR TITLE
Data visualisation

### PR DESCRIPTION
**Pull request by marcuspp**

### CODE REVIEW

**Comment by dvoong** [comment]
tests for the views will follow once we fix our sso testing issues

File: app/api/views.py [hunk 1]
```diff
@@ -271,6 +272,124 @@ def populate_database():
...
 
 
+@api.route('/api/v1/get-data-visualisation-data/<field>')
+@login_required
+def data_visualisation_data(field):
+    date_trunc = request.args.get('date_trunc', 'quarter')
+    exporter_status = request.args['exporter-status']
+    interests_table = internal_models.CountriesAndSectorsInterest.__tablename__
+
+    assert field in ['country', 'sector'], f'invalid field: {field}'
```
Comment (marcuspp) [inline app/api/views.py]: Not a fan of having asserts in code. What happens if when it does?

File: app/static/data_visualisation.js [hunk 2]
```diff
@@ -0,0 +1,832 @@
...
+                date: date,
+                dates: dates,
+                groups: groups,
+                groupColourScale: groupColourScale,
+            }
+        );
+    }
+
+    setDate(date) {
+        console.log("setDate");
```
Comment (marcuspp) [inline app/static/data_visualisation.js]: Are these still needed?

**Comment by marcuspp** [comment]
Looks good :+1:

File: app/api/views.py (hunk 1 above)
Comment (dvoong) [reply app/api/views.py]: udpated

File: app/static/data_visualisation.js (hunk 2 above)
Comment (dvoong) [reply app/static/data_visualisation.js]: I suspect I'll have to revisit these visualisations and these will be helpful for tracking the state until I learn to use that cool react tool that was demo'd in the show and tell a few weeks back

**Pull request merged**